package-lock.json

```diff
@@ -0,0 +1,10688 @@
+{
```

Comment:
I think we recently removed `package-lock.json` from the repo so it would get regenerated every time someone ran the bundler.

Reply:
Ack. I removed it to see how that would behave.